packages/react/readme.md

```diff
@@ -1,37 +1,42 @@
-# `@mdx-js/tag`
+# `@mdx-js/react`
 
 [![Build Status][build-badge]][build]
 [![lerna][lerna-badge]][lerna]
 [![Join the community on Spectrum][spectrum-badge]][spectrum]
 
 Map components to HTML elements based on the Markdown syntax.
-Useful with [MDX][].
+Serves as the React implementation for [MDX][].
 
 ## Installation
 
 [npm][]:
 
 ```sh
-npm i -S @mdx-js/tag
+npm install --save @mdx-js/react
 ```
 
 ## Usage
 
 ```jsx
+/* @jsx mdx */
 import React from 'react'
 import {renderToString} from 'react-dom/server'
-import {MDXTag} from '@mdx/tag'
+import {MDXProvider, mdx} from '@mdx/react'
```

Comment:
Should you be able to import `mdx` here? Doesn't look like it implemented if that's the case